packages/oui-select-picker/select-picker.less

```diff
@@ -8,3 +8,32 @@
   #oui > .select-picker(@oui-select-picker-selector);
   #oui > .select-picker-status();
 }
+
+.oui-select-picker_light {
+  &:not(.oui-select-picker_error) {
```

Comment:
What if we want the support of light variant with error ?
Manage your CSS priorities, this will not be a problem :)